.github/fabricbot.json

```
@@ -3747,6 +3703,45 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssuesOnlyResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "labelAdded",
+              "parameters": {
+                "label": "help wanted"
+              }
+            },
+            {
+              "name": "isAction",
+              "parameters": {
+                "action": "labeled"
+              }
+            }
+          ]
+        },
+        "eventType": "issue",
+        "eventNames": [
+          "issues",
+          "project_card"
+        ],
+        "taskName": "Drop a comment when a help candidate issue is identified",
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Looks liek this issue has been identified as a candidate for community contribution. If you're a community member, considering to send a PR for this issue, look for the `Summary Comment` link in the issue description. That comment has been left by an engineer on our team to help you get started with handling this issue. It contains high level guidance about what what approach you can take to address this issue as well as references to some useful relevant material. Good luck with your endeavour and we're looking forward to your PR. You can learn more about our Help Wanted process [here](https://aka.ms/aspnet/processes/help-wanted)"
```

Actually I trap myself regularly to don't read automated comments, because once read it's always the same, especially if the comment is quite long.

So the shorter, the better. Thus maybe just write it as:
```suggestion
              "comment": "Looks liek this issue has been identified as a candidate for community contribution. If you're considering to send a PR for this issue, look for the `Summary Comment` link in the issue description. That comment has been left by an engineer on our team to help you get started with handling this issue. You can learn more about our Help Wanted process [here](https://aka.ms/aspnet/processes/help-wanted)"
```
Should be contain all relevant info, and is short enough to be read.

The removed part
> That comment has been left by an engineer on our team to help you get started with handling this issue. It contains high level guidance about what what approach you can take to address this issue as well as references to some useful relevant material.

could be added to https://github.com/dotnet/aspnetcore/blob/main/docs/HelpWantedProcess.md as intro.